packages/components/src/internal/components/samples/models.ts

```diff
@@ -38,37 +39,31 @@ export const INDEPENDENT_SAMPLE_CREATION: SampleCreationTypeModel = {
     quantityLabel: 'New Samples',
 };
 
-export const CHILD_SAMPLE_CREATION: SampleCreationTypeModel = {
-    type: SampleCreationType.FromSources,
-    description: 'Create multiple output samples per source.',
-    minParentsPerSample: 1,
-    iconSrc: 'derivatives',
-    quantityLabel: 'New Samples per Source',
-};
-
 export const DERIVATIVE_CREATION: SampleCreationTypeModel = {
     type: SampleCreationType.Derivatives,
-    description: 'Create multiple output samples per parent.',
+    description: 'Create samples of different types from each selected sample.',
```

Comment:
the "of different types" part isn't yet implemented, but I assume this is just a "change it once so we don't forget" change?

Reply:
Yes, but it's also potentially "of a different type" (singular) for now, so seems OK to leave it.